packages/devtools_app/lib/src/shared/dialogs.dart

```diff
@@ -54,23 +53,25 @@ class DialogWithDefaults extends StatelessWidget {
   @override
   Widget build(BuildContext context) {
     return DevToolsDialog(
-      title: _DialogWithDefaultsTitle(onResetDefaults: onResetDefaults),
+      title: _StateUpdateDialogTitle(onResetDefaults: onResetDefaults),
       content: Container(
         padding: const EdgeInsets.symmetric(
           horizontal: defaultSpacing,
         ),
         width: dialogWidth ?? defaultDialogWidth,
-        child: Column(
-          mainAxisSize: MainAxisSize.min,
-          crossAxisAlignment: CrossAxisAlignment.start,
-          children: [
-            child,
-            if (helpText != null) ...[
-              const SizedBox(height: defaultSpacing),
-              DialogHelpText(helpText: helpText!),
-              const SizedBox(height: defaultSpacing),
+        child: Padding(
+          padding: const EdgeInsets.only(bottom: defaultSpacing),
```

Comment:
we don't need an extra padding widget here. Use the padding property on Container above